src/main.ts

```diff
@@ -87,6 +87,7 @@ export type Options = {
   lowerCaseServiceMethods: boolean;
   nestJs: boolean;
   env: EnvOption;
+  timestampAsString: boolean;
```

Comment:
Hm. Given we already have `forceLong` which can be `long` or `string`, wonder if we should have (the newly added) `useDate` be something like `builtin` (to use `Date`) / `string` (as you propose) / `protobuf` (the `google.protobuf.Timestamp` type as-is).

I don't really love those names (`builtin` / `string` / `protobuf`)...ideally could think of better names.

But the main idea would be to reuse the existing `useDate` flag, by turning it into a tri-state.

Maybe just:

* `useDate=true` -- default behavior, uses `Date`
* `useDate=false` -- as-is today, uses `google.protobuf.Timestamp
* `useDate=string` -- as you want to add